### DISCUSSION TITLE
How to disable q-slider's transition animation?

### DISCUSSION THREAD

**Discussion by chiwa737** [quasarframework/quasar#14620] [General - Components / Directives / etc] [answered]
In out application we need to have good respond UI, so that user feels latency as little as possible. While q-slider's transition transform blocks the way, it introduces (several hundreds milliseconds delay), I did not find any property to control it.

I also can see in quasar\s code that it has Qslide.sass that defines its outlook and some behavior, but I do not know how to override it by my sass code.

**Comment by pdanpdan** [comment, marked as answer]
make a codepen

**Comment by chiwa737** [comment]
Here is a codepen,  https://codepen.io/chiwa737/pen/zYjegjR

**Reply by pdanpdan** [reply]
check the styles

https://codepen.io/pdanpdan/pen/VwxRZXv

**Reply by jaybo** [reply]
This was super helpful!  If you have a slider that's animating its position with time (not through user interaction), you can eat up multiple frames with the slider animation, and this totally removes that overhead!
